<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Donors</class>
 <widget class="QDialog" name="Donors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Donors</string>
  </property>
  <widget class="QLineEdit" name="txtFirstName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMiddleName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLastName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtAge">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDateofBirth">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtGender">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtAreaofResidence">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>500</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtOccupation">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtContactNumber">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>160</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEmailAddress">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>210</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDateDonated">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>270</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBloodType">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>320</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtIdentification">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>380</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShow">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
image: url(&quot;C:/Users/clint/Pictures/sblood.png&quot;);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>SureNeed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Donor's Personal Record</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
